lib/utils.js

```diff
@@ -2,14 +2,17 @@ var Promise = require('bluebird');
 var deepCopy = require('deep-copy');
 var traverse = require('traverse');
 
-module.exports.clone = function(obj, preserveFunctions){
+module.exports.clone = function(obj, circular){
```

Comment:
PreserveFunctions was a leftover thing anyhow - I don't think it was used
anywhere

On Friday, April 15, 2016, Simon Bishop <simon@tenacious.digital> wrote:

> Only an argument name has changed - I left deep copy there so we have the
> possibility of cloning circular refs - this attention to detail is great!
>
> On Friday, April 15, 2016, Johan <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> In lib/utils.js
>> <https://github.com/happner/happn/pull/66#discussion_r59932034>:
>>
>> > @@ -2,14 +2,17 @@ var Promise = require('bluebird');
>> >  var deepCopy = require('deep-copy');
>> >  var traverse = require('traverse');
>> >
>> > -module.exports.clone = function(obj, preserveFunctions){
>> > +module.exports.clone = function(obj, circular){
>>
>> This function's prototype has changed, but I cannot see any references to
>> it that has changed accordingly.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/happner/happn/pull/66/files/3bf0fa902489a46e3ed9061f834ce34863b2b1f9#r59932034>
>>
>
>
> --
>
> SIMON BISHOP
>
> DIRECTOR
>
> +27 (0) 83 282 0703
>
> |     simon@tenacious.digital
> <javascript:_e(%7B%7D,'cvml','brianna@tenacious.digital');>
>
> twitter: @_tenacious_ <http://www.twitter.com/_tenacious_>
>
> |     skype: sjbishop
>
> 201 Salt Circle Arcade, Salt River, Cape Town, 7925
>
> www.tenacious.digital
>
>

-- 

SIMON BISHOP

DIRECTOR

+27 (0) 83 282 0703

|     simon@tenacious.digital <brianna@tenacious.digital>

twitter: @_tenacious_ <http://www.twitter.com/_tenacious_>

|     skype: sjbishop

201 Salt Circle Arcade, Salt River, Cape Town, 7925

www.tenacious.digital